src/main/resources/Quests.json

```diff
@@ -127,5 +132,335 @@
       ],
       "questPointsReqs":0,
       "questPointsReward":1
+   },
```

Comment:
Need to add empty skill reqs here? @apan145 or will it work if the field is not there if not needed.

Reply:
It's there already? Above the quest point requirements.

Reply:
Yup, i'm blind/editor compressed it perfectly.